tests/test_barostat.py

```diff
@@ -80,10 +82,16 @@ def test_barostat_with_clashes():
     mol_a, _, _ = get_hif2a_ligand_pair_single_topology()
     ff = Forcefield.load_from_file("smirnoff_1_1_0_sc.py")
 
-    # Set the lambda to 0.0 and don't minimize, resulting in clashes in the system
-    unbound_potentials, sys_params, masses, coords, box = get_solvent_phase_system(
-        mol_a, ff, lamb=0.0, minimize_energy=False
-    )
+    # Construct water box without removing the waters around the ligand to ensure clashes
+    host_system, host_coords, box, host_top = build_water_system(3.0, ff.water_ff)
+    # Shrink the box to ensure the energies are NaN
+    box -= np.eye(3) * 0.1
```

Comment:
With the box sizes being slightly larger now due to the additional waters on the edge of the box, the energy is now -2e8 rather than NaN. Reducing the box takes it back to Nan. The test passes either way to but just to be sure the energies are NaN I reduced the box as well.

 
Don't think it particularly matters as really the test is just to verify that if the system is undefined (NaN) that we won't make any barostat moves.